projects/dendrites/permutedMNIST/experiments/cns2021_figure1c_data/kw_sparsity_search.csv

```diff
@@ -0,0 +1,81 @@
+,Experiment name,Activation sparsity,FF weight sparsity,Dendrite weight sparsity,Num segments,Dim context,Epochs,Num tasks,LR,Momentum,Seed,Accuracy,ID
+0,sparsity_search,0.01,0.5,0.0,10,784,3,10,0.0005,0.0,2772,0.9366399645805359,sparsity_search kw_percent_on=0.01
+1,sparsity_search,0.01,0.5,0.0,10,784,3,10,0.0005,0.0,7630,0.9363999962806702,sparsity_search kw_percent_on=0.01
+2,sparsity_search,0.01,0.5,0.0,10,784,3,10,0.0005,0.0,9705,0.9332499504089355,sparsity_search kw_percent_on=0.01
```

Comment:
I don't think we need to add any CSV files, since the code itself can generate all CSV output.

Reply:
This applies to pkl files too.